### PR TITLE
Landing page text review

### PR DESCRIPTION
**Pull request by LLocci**
Review of landing page : 
- Make titles and subsections more coherent in the writing style
- Reformulate some stuffs

### CODE REVIEW

**Review by souksou — APPROVED** [review]

File: pages/index.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ export default function Home() {
       <div className="p-8 sm:p-28">
         <div className="font-bold text-lg">Features</div>
         <div className="max-w-screen-sm pb-16 text-black text-4xl font-bold">
-          <h2>Making people and actors of the industry the owner of their data.</h2>
+          <h2>Making individuals and industries the sole owners of their data.</h2>
```
Comment (AlexikM) [inline pages/index.tsx]: - I like "people"
- industries is not really an entity which can have ownership

File: pages/index.tsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ export default function Home() {
               alt="Dashed eye representing data privacy."
             />
             <div className="mt-4 mb-2 text-black text-xl font-bold">End-to-end privacy</div>
-            <div>Trustless to intermediaries as data is encrypted for specific receivers.</div>
+            <div>With end-to-end encryption, there is no need to trust intermediary actors.</div>
```
Comment (AlexikM) [inline pages/index.tsx]: The other way around, we want to emphasis on the result not the how, e.g. :
There is no need to trust intermediaries, as the data is protected from end-to-end...

File: pages/index.tsx [hunk 3]
```diff
@@ -88,7 +88,7 @@ export default function Home() {
             />
             <div className="mt-4 mb-2 text-black text-xl font-bold">Certified data</div>
             <div>
-              The origin and completeness of the data is tampered and can always be verified.
+            The origin and completeness of the data is tracked by design and can be verified at any time.
```
Comment (AlexikM) [inline pages/index.tsx]: try to include "traceability" keyword rather than "tracking"?

File: pages/index.tsx [hunk 4]
```diff
@@ -99,12 +99,12 @@ export default function Home() {
               alt="Dashed eye representing data privacy."
             />
             <div className="mt-4 mb-2 text-black text-xl font-bold">Distributed network</div>
-            <div>Connecting objects, people and organizations while ensuring independance.</div>
+            <div>Objects, people and organizations are interconnected but remain self-reliant.</div>
```
Comment (AlexikM) [inline pages/index.tsx]: Not a big fan of "Self-reliant", we talk to the basic english speaker

File: pages/index.tsx [hunk 5]
```diff
@@ -99,12 +99,12 @@ export default function Home() {
...
           </div>
           <div className="mb-16">
             <Image width={60} height={60} src={merge} alt="Dashed eye representing data privacy." />
             <div className="mt-4 mb-2 text-black text-xl font-bold">Performant editing</div>
-            <div>Excelling at data synchronization using a dynamic data structure.</div>
+            <div>Data synchronisation is optimized by using efficient dynamic data structures.</div>
```
Comment (AlexikM) [inline pages/index.tsx:109]: optimized + efficient has the same meaning, we can remove efficient in my opinion

File: pages/index.tsx [hunk 6]
```diff
@@ -117,7 +117,7 @@ export default function Home() {
       <div className="p-8 sm:p-28">
         <div className="font-bold text-lg">Industry</div>
         <div className="max-w-screen-sm pb-16 text-black text-4xl font-bold">
-          <h2>Protecting the data itself to move it from silos towards flexible networks.</h2>
+          <h2>Moving from data silos to flexible networks by securing data constitutionally.</h2>
```
Comment (AlexikM) [inline pages/index.tsx]: not a big fan of constitutionally, doesn't speak to the basic english speaker I am.

Maybe security is not the first argument here, I would rethink this end of this title, we move from silos, to networks where we can share and collaborate on data...

File: pages/index.tsx [hunk 7]
```diff
@@ -134,7 +134,7 @@ export default function Home() {
           <div className="relative h-96 bg-black rounded overflow-hidden">
             <Image layout="fill" objectFit="cover" src={legal} alt="People signing a contract." />
             <div className="text-white text-2xl absolute bottom-0 left-0 p-4 bg-gradient-to-t from-black to-transparent leading-snug">
-              Certification for compliance and regulatory
+            Certifying data and complying with regulations
```
Comment (AlexikM) [inline pages/index.tsx]: perfect!

File: pages/index.tsx [hunk 8]
```diff
@@ -145,7 +145,7 @@ export default function Home() {
               alt="City from night exposed to attacks."
             />
             <div className="text-white text-2xl absolute bottom-0 left-0 p-4 bg-gradient-to-t from-black to-transparent leading-snug">
-              Cyber-defence to protect privacy and for security
+              Protecting privacy and populations with cyber-defense
```
Comment (AlexikM) [inline pages/index.tsx:150]: Let's rephrase like the previous one, e.g. : 
Defending data privacy and business secrets...
We can remove cyber-defense, we will make it appear somewhere else

File: pages/index.tsx [hunk 9]
```diff
@@ -156,7 +156,7 @@ export default function Home() {
               alt="Image of the earth representing communities from multiple cities which generate big data."
             />
             <div className="text-white text-2xl absolute bottom-0 left-0 p-4 bg-gradient-to-t from-black to-transparent leading-snug">
-              Processing big data and collaborating
+            Handling big data and fostering digital collaboration
```
Comment (AlexikM) [inline pages/index.tsx]: Scaling and processing big data?
Lets rethink this one

**Review by AlexikM — CHANGES_REQUESTED** [review]
Thanks a lot, lets rework on it to finalize

File: pages/index.tsx (hunk 5 above)
Comment (LLocci) [reply pages/index.tsx:109]: Hum, justement efficience is the explanation why we are optimized. dynamic data structures are not optimized by design.

File: pages/index.tsx (hunk 8 above)
Comment (LLocci) [reply pages/index.tsx:150]: why ? I do not see the problem here

**Pull request merged**